scratch/evaluate_scheduler_qd_dense_scenario.cc

```diff
@@ -0,0 +1,825 @@
+/*
```

Comment:
i only checked this scratch script, the same comments apply for all other scripts